xml/System.Security.Cryptography.Pkcs/SignerInfo.xml

```diff
@@ -593,9 +605,24 @@
         <Parameter Name="asnEncodedData" Type="System.Security.Cryptography.AsnEncodedData" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="asnEncodedData">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="asnEncodedData">The attribute to remove, an object with Abstract Syntax Notation One (ASN.1)-encoded data.</param>
```

Comment:
I feel like expanded name is not super useful - I wouldn't expect this API to be used by someone who doesn't know what ASN.1 means

Reply:
```suggestion
        <param name="asnEncodedData">The ASN.1 encoded attribute to remove from the document.</param>
```